src/Homepage/ProjectBarChart.jsx

```diff
@@ -52,7 +52,7 @@ class ProjectBarChart extends React.Component {
             <Tooltip />
             <Legend />
             <Bar dataKey="caseCount" stackId="a" fill="#8884d8" />
-            <Bar dataKey={tor.translate('experimentCount')} stackId="a" fill="#82ca9d" />
+            <Bar dataKey="coreVisitCount" stackId="a" fill="#82ca9d" />
```

Comment:
should still be translate for experimentCount and let coreVisit be the translated result for ndh